api/test/test_play_asset_pack_manager.gd

```diff
@@ -0,0 +1,47 @@
+# ##############################################################################
+#
+#	Copyright 2020 Google LLC
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		https://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+#
+# ##############################################################################
+extends "res://addons/gut/test.gd"
+
+func create_play_asset_pack_manager(mock_plugin):
+	var test_object = partial_double("res://src/play_asset_pack_manager.gd").new()
```

Comment:
[Partial Double](https://github.com/bitwes/Gut/wiki/Partial-Doubles) used here since we need to stub the ```_initialize_plugin```